frontend/src/components/MapChat/NewMessage.js

```diff
@@ -41,7 +42,7 @@ class NewMessage extends Component {
           title="Emoji"
         />
         <div className="extra-message-options">
-          <span className="emoji-picker-button" onClick={this.toggleEmojiPicker}>🙃</span>
+          <span className="emoji-picker-button" onClick={this.toggleEmojiPicker}><Emoji size={24} emoji="upside_down_face" /></span>
```

Comment:
yes, the big issue is rendering it inside a textarea/message

textarea is much harder, but even inside a message div, how do you turn

"hello there 😛  more text 🇺🇸 and more"

into a renderable array

"hello there ",
<Emoji ... />,
" more text ",
<Emoji ... />,
" and more"